gitlab/const.py

```diff
@@ -27,17 +30,48 @@
 MAINTAINER_ACCESS: int = 40
 OWNER_ACCESS: int = 50
 
+
+class AccessLevel(Enum):
+    NO_ACCESS: int = 0
+    MINIMAL_ACCESS: int = 5
+    GUEST_ACCESS: int = 10
+    REPORTER_ACCESS: int = 20
+    DEVELOPER_ACCESS: int = 30
+    MAINTAINER_ACCESS: int = 40
+    OWNER_ACCESS: int = 50
```

Comment:
If we're able to scope them into a class, should we follow GitLab's own convention for this?

```suggestion
class AccessLevel(Enum):
    NO_ACCESS: int = 0
    MINIMAL_ACCESS: int = 5
    GUEST: int = 10
    REPORTER: int = 20
    DEVELOPER: int = 30
    MAINTAINER: int = 40
    OWNER: int = 50
```

See https://gitlab.com/gitlab-org/gitlab/-/blob/e97357824bedf007e75f8782259fe07435b64fbb/lib/gitlab/access.rb#L12-18